test/Sets/SimpleSparsePolynomialZonotope.jl

```diff
@@ -0,0 +1,19 @@
+@testset "Simple Sparse Polynomial Zonotope" begin
```

Comment:
We usually put the `@testset` into `runtests.jl` and instead use a loop for different number types. Check one of the other test files.

Also, this file is not included in `runtests.jl` and hence these tests are not executed.